### ISSUE TITLE
Accessibility Review

### ISSUE THREAD

**Issue by josephmaxwellhyland** [hto-projects/hytop#261]
We should do a review of the accessibility of HyTOP holistically, to make sure it meets all the web accessibility standards. From Duncan Roberts:

>Hi. Quick recommendation. I was just playing around with HyTOP and it would be a really good learning opportunity if the future interns working on it do some basic accessibility testing. They could install the Accessibility Insights for Web plugin and just run some of the guided tests to see what's what. I didn't do a lot of testing, but there are some accessibility issues like funky tab orders and missing alt text that are easy to see and fix. It would be a great learning opportunity because software accessibility isn't going anywhere and anyone going into software development in the future will need to know about it.
